ui-components/src/page_components/listing/listing_sidebar/SidebarAddress.tsx

```diff
@@ -1,13 +1,12 @@
 import * as React from "react"
 import ReactDOMServer from "react-dom/server"
 import { Icon, IconFillColors } from "../../../icons/Icon"
-import { Address } from "@bloom-housing/backend-core/types"
```

Comment:
These changes were not necessary, just removing references to our backend from ui-components when I'm touching these components